src/components/AttachmentModal.js

```diff
@@ -169,17 +167,6 @@ class AttachmentModal extends PureComponent {
      * @returns {Boolean}
      */
     isValidFile(file) {
-        const {fileExtension} = FileUtils.splitExtensionFromFileName(lodashGet(file, 'name', ''));
```

Comment:
If we are going to implement a disallowed list of file extensions on the server, the frontend should do the same. Let's HOLD on this until we know if we're going to do that yet or not.